pom.xml

```diff
@@ -91,7 +91,7 @@
         <version.plugin.compiler>3.8.1</version.plugin.compiler>
         <version.plugin.dependency>3.1.2</version.plugin.dependency>
         <version.plugin.directory>0.1</version.plugin.directory>
-        <version.plugin.eclipselink>2.7.1.1</version.plugin.eclipselink>
+        <version.plugin.eclipselink>2.7.5.1</version.plugin.eclipselink>
```

Comment:
Should also update `applications/mp/pom.xml` and `examples/quickstarts/helidon-standalone-quickstart-mp/pom.xml`